sklearn/model_selection/_search.py

```diff
@@ -1414,12 +1414,14 @@ class RandomizedSearchCV(BaseSearchCV):
 
     Examples
     --------
+    >>> import numpy as np
     >>> from sklearn.datasets import load_iris
     >>> from sklearn.linear_model import LogisticRegression
     >>> from sklearn.model_selection import RandomizedSearchCV
     >>> from scipy.stats import uniform
     >>> iris = load_iris()
     >>> logistic = LogisticRegression(solver='saga', tol=1e-2, max_iter=200)
+    >>> np.random.seed(seed=233423)  # make sure uniform will be deterministic
```

Comment:
That would be global I think? Shouldn't we pass a random state to `LogisticRegression` in addition to the `GridSearchCV`?

Reply:
We could but I am not sure that this is actually the issue. I think it should be linked to the uniform for which the random state cannot be set

Reply:
Uhm locally, I get determimistic results with master. I see that the behaviour of the random state could be an issue but this is for SciPy < 0.16 which is not the case with the failing build.